src/books.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  # ...
+
+  attr_reader :rentals
+
+  def initialize(_title, _author)
+    # ...
+    @rentals = []
+  end
```

Comment:
- [ ] the @title and @author instance variables should be initialized in the constructor. please find a link to the example about constructors in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars
- [ ] there should be a setter and getter created for these instance variables as well.